system/include/emscripten/val.h

```diff
@@ -563,7 +563,7 @@ namespace emscripten {
             return internal::_emval_delete(handle, val(property).handle);
         }
 
-        void throw_() const {
+        [[noreturn]] void throw_() const {
```

Comment:
I guess its OK to use this over `__attribute__((noreturn))` since embind probably only supports C++11 and above anyway?

Reply:
Yeah.

Reply:
It would be good to check that. I'm not sure myself after looking around a little - nothing obvious in the code suggests C++11 is required. libembind itself is compiled without any C++ version flags, in fact.

The safe thing is probably to use the attribute.

Reply:
This file has an assert at the top that it is C++11 so I think we are good.

Reply:
Oh, good catch!